netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -340,6 +345,34 @@ _ebpf_sock_addr_get_socket_cookie(_In_ const bpf_sock_addr_t* ctx)
     return sock_addr_ctx->transport_endpoint_handle;
 }
 
+static uint32_t
+_ebpf_sock_addr_get_interface_type(_In_ const bpf_sock_addr_t* ctx)
+{
+    net_ebpf_sock_addr_t* sock_addr_ctx = CONTAINING_RECORD(ctx, net_ebpf_sock_addr_t, base);
```

Comment:
I think these helper functions should check which hook they are being invoked for and return an error code to explicitly tell the BPF program that they incorrectly invoked the helper function.